aeron-client/src/main/java/io/aeron/CncFileDescriptor.java

```diff
@@ -142,17 +149,18 @@ public class CncFileDescriptor
         CLIENT_LIVENESS_TIMEOUT_FIELD_OFFSET = ERROR_LOG_BUFFER_LENGTH_FIELD_OFFSET + SIZE_OF_INT;
         START_TIMESTAMP_FIELD_OFFSET = CLIENT_LIVENESS_TIMEOUT_FIELD_OFFSET + SIZE_OF_LONG;
         PID_FIELD_OFFSET = START_TIMESTAMP_FIELD_OFFSET + SIZE_OF_LONG;
+        FILE_PAGE_SIZE_FIELD_OFFSET = PID_FIELD_OFFSET + SIZE_OF_LONG;
     }
 
     /**
      * Length of the metadata header for the CnC file.
      */
-    public static final int META_DATA_LENGTH = PID_FIELD_OFFSET + SIZE_OF_LONG;
+    public static final int META_DATA_LENGTH = CACHE_LINE_LENGTH * 2;
```

Comment:
I'm wondering if this needed to change?  We do lose having a single constant that represents the unaligned length of the cnc metadata.  I'm not sure if this is a big drawback or not.

The Javadoc should be updated to reflect that it is the aligned length of the metadata.

Reply:
It this to ensure consistent implementation between C & Java?

Reply:
Yes, it was aligning C and Java impl. The former only had full aligned length.